### PR TITLE
[FEATURE] MS-31: Interviews with redux

### PR DESCRIPTION
**Pull request by EugenioPonzone**
CRUD interviews with REDUX.

### CODE REVIEW

File: src/Components/Interviews/index.jsx [hunk 1]
```diff
@@ -4,79 +4,57 @@ import List from './List';
...
+    return () => {
+      dispatch(interviewsCleanUp());
+    };
   }, []);
 
   //MODAL
   const closeModal = () => {
-    setShowModal(false);
+    dispatch(hideModal());
   };
```
Comment (BaltazarCortina) [inline src/Components/Interviews/index.jsx:32]: Same as applications

File: src/Components/Interviews/index.jsx [hunk 2]
```diff
@@ -4,79 +4,57 @@ import List from './List';
...
     <>
-      <Modal
-        showModal={showModal}
-        closeModalFn={closeModal}
-        acceptModalFn={acceptModal}
-        content={selectedItem}
-        type={typeModal}
-        titleModal={titleModal}
-      />
+      {modal && <Modal closeModalFn={closeModal} acceptModalFn={acceptModal} />}
```
Comment (BaltazarCortina) [inline src/Components/Interviews/index.jsx:48]: Same as applications

File: src/redux/interviews/thunks.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        dispatch(showModal('interviews', 'error', err.message));
+      });
+  };
+};
+
+//DELETE INTERVIEW
+export const deleteInterview = (id) => {
+  return (dispatch) => {
+    dispatch(deleteInterviewFetching());
+    dispatch(updateModal('fetching', { info: 'Loading...' }));
```
Comment (BaltazarCortina) [inline src/redux/interviews/thunks.js:75]: Same as applications

**Review by Sabrucita — APPROVED** [review]
As Balti says 😋

**Review by BaltazarCortina — APPROVED** [review]

**Pull request merged**